.github/workflows/lvs_gf180mcu_fd_sc_test.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a Github Actions for testing LVS rule deck of GF180MCU
+name: Standard Cells (gf180mcu_fd_sc_mcu7t5v0 and gf180mcu_fd_sc_mcu9t5v0) LVS testing
```

Comment:
Can you split the standard cell libraries into separate runs?

Reply:
Done